candis/data/entrez/api.py

```diff
@@ -8,72 +14,180 @@
 def sanitize_response(response, type_ = 'json'):
     if type_ == 'json':
         data  = response.json()
-        kres  = data['header']['type'] + 'result'
-
-        data  = data[kres]
+        try:
+            kres  = data['header']['type'] + 'result'
+            data = data[kres]
+        except KeyError:
+            kres = 'result'  # for esummary
+            data = data[kres]
     else:
         # TODO: What other types can be handled? XML, HTML, etc.
         pass
 
     return data
 
+def sanitize_term(term):
+    if isinstance(term, list) and term:
+        term = [t.replace(' ','') for t in term]
+        return ' AND '.join(term)
+    else:
+        raise TypeError('term should be a list with atleast 1 item')
+
+def params_dict2string(params):
+    if isinstance(params, dict):
+        return str(urlencode(params)).replace('%40', '@')
+    else:
+        raise TypeError('params should be a dictionary')
+
+
 # TODO: Should we cache each response?
 class API(object):
     # TODO: Assign CONFIG.NAME to NAME
```

Comment:
We need to add in config file, 'NAME' field, and then use it here ? @achillesrasquinha